pokemons/helpers.py

```diff
@@ -29,3 +41,32 @@ def pokemon_exists(name):
     if requests.get(url).status_code == 404:
         return False
     return True
+
+
+def get_pokemons_from_trainerteam(battle, trainer):
+    trainerteam = TrainerTeam.objects.get(
+        battle_related=battle,
+        trainer=trainer
+    )
+    return {
+        Pokemon.objects.get(name=trainerteam.pokemon_1),
```

Comment:
@lucabezerra I've just realized these queries wasn't needed, because I could get pokemons through the function local variable `trainerteam`.

About the docs, I read and tested with `User` and `TrainerTeam`, but I couldn't test with `Pokemon` and `TrainerTeam` (do you think having more than one instance of `Pokemon` in `TrainerTeam` affected it?)